### PR TITLE
[Method References 2] LPS-167246 Convert method reference to ServiceTracker in module portal-messaging

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by W2jx** [comment]
no conflicts(1/31 10:10)

**Comment by tinatian** [comment]
@W2jx This (https://github.com/liferay-core-infra/liferay-portal/pull/3201/commits/64cc1acc2dabf4ee40a6e22d93ba7a039bc196b0) is not an "SF", this is a "Fix typo"

**Comment by tinatian** [comment]
@W2jx Please separate this into two https://github.com/liferay-core-infra/liferay-portal/pull/3201/commits/a402662be0920cfc14acf061e969dae793367970

**Comment by W2jx** [comment]
Hi @tinatian , I have resent it.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:lima

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/DefaultMessageBus.java [hunk 1]
```diff
@@ -478,19 +351,178 @@ private void _updateDestination(
 	private static final Log _log = LogFactoryUtil.getLog(
 		DefaultMessageBus.class);
 
+	private BundleContext _bundleContext;
+	private ServiceTracker<DestinationEventListener, DestinationEventListener>
+		_destinationEventListenerServiceTracker;
 	private final Map<String, Destination> _destinations =
```
Comment (tinatian) [inline modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/DefaultMessageBus.java:357]: @W2jx As you already have service tracker for destinations, you should not need this map here. Please use service tracker map for destination and remove it here.

**Pull request closed without merging**